File: Beecrowd/1 - Beginner/1101 - 1190/1182 - Column in Array/C/1182_ColumninArray.c
#include <stdio.h>

int main() {
    int column;
    char operation;
    double matrix[12][12], sum = 0.0;
    scanf("%d %c", &column, &operation);

    for (int i = 0; i < 12; i++) {
        for (int j = 0; j < 12; j++) {
            scanf("%lf", &matrix[i][j]);

            if (j == column) {
                sum += matrix[i][j];
            }
        }
    }

    if (operation == 'S') printf("%.1lf\n", sum);
    else printf("%.1lf\n", sum / 12.0);

    return 0;
}

File: Beecrowd/1 - Beginner/1101 - 1190/1182 - Column in Array/Python/1182_ColumninArray.py
column = input()
operation = input()
column = int(column)
sum_val = 0.0

for i in range(12):
    for j in range(12):
        number = float(input())
        if j == column:
            sum_val += number

if operation == 'S':
    print(f"{sum_val:.1f}")
else:
    print(f"{sum_val / 12.0:.1f}")
